bindings/python/manifold3d.cpp

```diff
@@ -315,6 +315,16 @@ NB_MODULE(manifold3d, m) {
       .def("calculate_curvature", &Manifold::CalculateCurvature,
            nb::arg("gaussian_idx"), nb::arg("mean_idx"),
            manifold__calculate_curvature__gaussian_idx__mean_idx)
+      .def("calculate_normals", &Manifold::CalculateNormals,
+           nb::arg("normal_idx"), nb::arg("min_sharp_angle"),
+           manifold__calculate_normals__normal_idx__min_sharp_angle)
+      .def(
+          "smooth",
+          [](const Manifold &self, float minSharpAngle, float minSmoothness) {
+            return self.Smooth(minSharpAngle, minSmoothness);
```

Comment:
@pca006132 Alas, no luck here. I wish the error wasn't such a pile of spew. Also, even when I build with MANIFOLD_PYBIND=ON I don't get the error - is something else required to make it build?

Reply:
weird... will debug this

Reply:
It seems that nanobind is unhappy with `smooth` being both a static and instance method for Manifold`.

```
Critical nanobind error: nb::detail::nb_func_new("smooth"): mismatched static/instance method flags in function overloads!
```

Reply:
Okay, I'm going with `SmoothOut` - slightly more verb sounding.